packages/core/src/Stores/client-store.js

```diff
@@ -1431,7 +1431,7 @@ export default class ClientStore extends BaseStore {
 
     get residence() {
         if (this.is_logged_in) {
-            return this.account_settings.country_code ?? '';
+            return this.accounts[this.loginid].country ?? '';
```

Comment:
Just a question, is there any instance where we get this `residence` before `this.accounts[this.loginid]` is set?